src/view/Draw.java

```diff
@@ -1,171 +1,26 @@
 package view;
 
-import java.awt.Color;
-import java.awt.Point;
-import java.util.ArrayList;
-
-import model.*;
-import model.specific_path.Circle;
-import model.specific_path.LineType;
-import model.specific_path.Path;
-import model.specific_path.PathPart;
-import model.specific_path.Rectangle;
+import language.BasicPainter;
+import model.presentation.awt.AWTFrame;
 import model.presentation.xml.SVGFile;
 
 public class Draw {
 
 	public static void main(String[] args) {
-		
-		/************************************
-		 *	Circle and Rectangle
-		 ************************************/
-		
-		// Create a new black pencil
-		Pencil p = new Pencil(4, new ColorSVG(Color.black.getRGB()));
-		
-		// Create a Cirle with a radius of 30 with the center in (50,40)
-		Circle c = new Circle(50,40,30);
-		
-		// Fill the center of the circle in blue
-		c.setColor(new ColorSVG(Color.blue.getRGB()));
-		
-		// Add the pencil to the Circle
-		c.setPencil(p);
-		
-		// Create a Rectangle
-		Rectangle r = new Rectangle(55,45,50,50);
-		
-		// Set the pencil
-		r.setPencil(p);
-		
-		// Fill it
-		r.setColor(new ColorSVG(Color.green.getRGB()));
+		BasicPainter painter = new BasicPainter();
+		Painting painting = new Painting();
+		painting.draw(painter);
 
-		// Gather all the shapes

```

Comment:
Maybe keep the two way of doing it, it might be more useful if we have a problem with either one of them.

And you also removed the comment without adding new ones. Could you do that ?